apps/mobile/src/components/InsideRouter/atoms.ts

```diff
@@ -1,13 +1,22 @@
 import {atom} from 'jotai'
 import {privateApiAtom} from '../../api'
 import * as TE from 'fp-ts/TaskEither'
+import * as T from 'fp-ts/Task'
 import {pipe} from 'fp-ts/function'
 import {toCommonErrorMessage} from '../../utils/useCommonErrorMessages'
 import {translationAtom} from '../../utils/localization/I18nProvider'
 import {type Task} from 'fp-ts/Task'
 import reportError from '../../utils/reportError'
 import {type GetCryptocurrencyDetailsResponse} from '@vexl-next/rest-api/dist/services/user/contracts'
 import showErrorAlert from '../../utils/showErrorAlert'
+import {Duration} from 'luxon'
+import {UnixMilliseconds} from '@vexl-next/domain/dist/utility/UnixMilliseconds.brand'
+
+const BTC_PRICE_UPDATE_TRIGGER_THRESHOLD_MINUTES = 15
+
+const btcPriceLastUpdateAtAtom = atom<UnixMilliseconds>(
+  UnixMilliseconds.parse(0)
```

Comment:
There is constant for this `UnixMilliseconds0` in `@vexl-next/domain/dist/utility/UnixMilliseconds.brand`.